### PR TITLE
Feat: Dart 3.1

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Starter for Dart 3.1

## Test Plan

None

## Related PRs and Issues

X

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: dart-3.1/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  // The `req` object contains the request data
+  if (context.req.method == 'GET') {
+    // Send a response with the res object helpers
+    // `res.send()` dispatches a string back to the client
+    return context.res.send('Hello, World!');
+  }
+
+  // `res.json()` is a handy helper for sending JSON
+  return context.res.json({
+    'motto': 'Build Fast. Scale Big. All in One Place.',
```
Comment (christyjacob4) [inline dart-3.1/lib/main.dart]:
```suggestion
    'motto': 'Build like a team of hundreds_',
```
Might wanna change this in all runtimes

File: dart-3.1/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- Returns a "Learn More" JSON response.
+
+**Response**
+
+Sample `200` Response:
+
+```json
+{
+  "motto": "Build Fast. Scale Big. All in One Place.",
```
Comment (christyjacob4) [inline dart-3.1/README.md]:
```suggestion
  "motto": "Build like a team of hundreds_",
```
Here as well

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Looks good. Just a small comment

File: dart-3.1/lib/main.dart (hunk 1 above)
Comment (Meldiron) [reply dart-3.1/lib/main.dart]: Updated everywhere

File: dart-3.1/README.md (hunk 2 above)
Comment (Meldiron) [reply dart-3.1/README.md]: Updated everywhere

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**